.gitignore

```diff
@@ -2,3 +2,4 @@
 node_modules
 phpunit.xml
 vendor
+.idea
```

Comment:
Please don't add that. This doesn't belong in here. It belongs in your global system ignore file.